datasafe-rest-impl/2.startDockerImageWithAmazon.sh.template

```diff
@@ -0,0 +1,36 @@
+#
+# trap with sigint because container can not be stopped from command line otherwise
+#
+
+trap stop SIGINT
```

Comment:
Instead of trap it is enough to add `-it` to run args if not to use redirection

Reply:
yes